tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -89,3 +89,36 @@ Feature: spaces management
       | team.c |
       | team.d |
     And "Alice" logs out
+
+
+  Scenario: list members via sidebar
+    Given "Admin" creates following users
+      | id      |
+      | Alice   |
+      | Brian   |
+      | Carol   |
+      | Marie   |
+      | Richard |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Space Admin |
+    And "Alice" creates the following project spaces using API
+      | name   | id     |
+      | team A | team.a |
+    And "Alice" adds following members to the space "team A" using API
+      | user    | role   | shareType |
+      | Brian   | editor | space     |
+      | Carol   | viewer | space     |
+      | Marie   | viewer | space     |
+      | Richard | viewer | space     |
```

Comment:
can we change here as "Admin" creates space and adds users to space?

we will thereby be able to verify that space admin can see another's users spaces

Reply:
That would be great but right now as we don't clear up spaces at the end of test execution and as the admin user is not recreated, this test might be flaky in CI if I change it to "Admin" creates space and adds users to space. Because locally as well I cannot run this test twice without deleting the space. So maybe we can do this after spaces are cleaned up in this issue https://github.com/owncloud/web/issues/8662?

Reply:
I've added a comment in the issue so we don't forget this https://github.com/owncloud/web/issues/8662#issuecomment-1480660127